build/rpm_metadata/requires

```diff
@@ -15,19 +15,19 @@ Requires: pupmod-simp-auditd < 7.0.0-0
 Requires: pupmod-simp-auditd >= 6.0.0-0
 Requires: pupmod-simp-autofs < 6.0.0-0
 Requires: pupmod-simp-autofs >= 5.0.0-0
-Requires: pupmod-simp-clamav < 6.0.0-0
+Requires: pupmod-simp-clamav < 7.0.0-0
 Requires: pupmod-simp-clamav >= 5.0.0-0
```

Comment:
Will this really interoperate with an older version of pupmod-simp-clamav, et. al?  I thought changes were required to add the environment paths to the rsync behavior in those modules?